myuw/templates/handlebars/notices.html

```diff
@@ -175,7 +175,7 @@ <h2 id="main" class="heading-page">Notices</h2>
                 {{ /if }}
                 <!-- end collapsy 3 -->
 
-                {{ #if next_week.notices }}
+                {{ #if next_week.tices }}
```

Comment:
typo?

Reply:
good catch. I didn't realize i even edited this page. Might have been a find and replace issue. Thanks!